stylesheets/components/CompositionArea.scss

```diff
@@ -55,6 +55,8 @@
     align-items: center;
     background: none;
     border: none;
+    width: 32px;
```

Comment:
Shouldn't the border and height be dynamic in some way?

Reply:
@ArgirisKatsoulieris This was a fix to make the send button line up with the other icons, all of which are fixed at 32px. It was due to the fact that [this PR](https://github.com/signalapp/Signal-Desktop/pull/6219/files) had not been merged yet. That PR has now been merged, so I believe this is an approved solution.